### PR TITLE
fix(agw): Freediameter build

### PR DESCRIPTION
**Pull request by gjalves**
fix(agw): Change path of patch file

## Summary

Change path of 0001-opencoord.org.freeDiameter.patch from lte/gateway/c/oai/patches/ to lte/gateway/c/core/oai/patches/

## Test Plan

When manually try to build freediameter using third_party/build/bin/freediameter_build.sh, the script fails with error

`./freediameter_build.sh: line 62: /home/gustavo-alves/devel/magma-gj//lte/gateway/c/oai/patches/0001-opencoord.org.freeDiameter.patch: No such file or directory`

After the patch, compilation works fine

### CODE REVIEW

**Comment by maxhbr** [comment]
good catch, but shouldn't this script also apply `0002-opencoord.org.freeDiameter.patch`?

**Comment by gjalves** [comment]
Thanks @maxhbr , you are right. I´ve just added and tested with the other patch.

**Review by maxhbr — APPROVED** [review]
LGTM

**Comment by gjalves** [comment]
I´m facing some permission trouble setting the label as ready2merge. Someone can do the favor? Thanks.

File: third_party/build/bin/freediameter_build.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ git clone "${GIT_URL}" freediameter
 cd freediameter
 git checkout "${GIT_COMMIT}"
 
-patch -p1 < $MAGMA_ROOT/lte/gateway/c/oai/patches/0001-opencoord.org.freeDiameter.patch
+patch -p1 < $MAGMA_ROOT/lte/gateway/c/core/oai/patches/0001-opencoord.org.freeDiameter.patch
```
[the remark this answers is not in the document]
Comment (maxhbr) [reply third_party/build/bin/freediameter_build.sh]:
```suggestion
patch -p1 < "$MAGMA_ROOT/lte/gateway/c/core/oai/patches/0001-opencoord.org.freeDiameter.patch"
```

File: third_party/build/bin/freediameter_build.sh [hunk 2]
```diff
@@ -59,7 +59,8 @@ git clone "${GIT_URL}" freediameter
...
+patch -p1 < $MAGMA_ROOT/lte/gateway/c/core/oai/patches/0002-opencoord.org.freeDiameter.patch
```
[the remark this answers is not in the document]
Comment (maxhbr) [reply third_party/build/bin/freediameter_build.sh]:
```suggestion
patch -p1 < "$MAGMA_ROOT/lte/gateway/c/core/oai/patches/0002-opencoord.org.freeDiameter.patch"
```

**Review by maxhbr — CHANGES_REQUESTED** [review]
shellcheck is right, you should add quotes to prevent word-splitting

**Review by maxhbr — APPROVED** [review]
LGTM

**Comment by Neudrino** [comment]
@gjalves: The DCO check is still failing. See the first comment on the PR on how to fix it, please.

**Comment by gjalves** [comment]
@Neudrino , I hope now everything should be fine.

**Pull request merged**